src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--1">
+      <div class="stars__star stars__star--active"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--2">
+      <div class="stars__star stars__star--active"></div>
+      <div class="stars__star stars__star--active"></div>
```

Comment:
```suggestion
    <div class="stars stars--1">
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
    </div>
    <div class="stars stars--2">
      <div class="stars__star"></div>
      <div class="stars__star"></div>
```
no need additional class for active stars.
for amount of active stars should be responsible stars--1, stars--2 ... stars--5  modifier. also use :nth-child() pseudo class.